tests/extmod/uzlib_decompress.py

```diff
@@ -42,7 +39,7 @@
 print(exp)
 # Even when you ask CPython zlib.compress to produce Raw DEFLATE stream,
 # it returns it with adler2 and oriignal size appended, as if it was a
-# zlib stream. Make sure there're no random issues decompressing such.
+# zlib stream. Make sre there're no random issues decompressing such.
```

Comment:
oops!

Reply:
that is codespell's mistake, I think!

Reply:
well, maybe not codespell, not sure how that happened. Just flailing fingers at some point. Tnx!

Reply:
I was guessing it was from global search and replace of `ure` to `re`, which resulted in `sure` changing to `sre`.